src/routes/WhoWeAre.svelte

```diff
@@ -16,7 +17,7 @@
         </p>
         <div class="w-full text-center md:text-left">
             <Button>
-                <span>See About</span>
+                <button on:click={() => goto('/about')}>See About</button>
```

Comment:
For accessibility reasons, let's not use `button` to emulate what an `<a href>` is already meant to do by default.

Reply:
Here is how I would refactor this:
```suggestion
                <a href="/about/"}>See About</button>
```